Increase the connector message size to avoid message to long errors from
uvesafb as per the below dmesg fragment:

uvesafb: Bochs/Plex86 Developers, Bochs/Plex86 VBE Adapter, $Id$, OEM: Bochs/Plex86 VBE(C) 2003 http://savannah.nongnu.org/projects/vgabios/, VBE v2.0
uvesafb: no monitor limits have been set, default refresh rate will be used
uvesafb: scrolling: redraw
uvesafb: framebuffer at 0xe0000000, mapped to 0xffffc90000080000, using 8192k, total 8192k
fb0: VESA VGA frame buffer device
uvesafb: message too long (57808), can't execute task
uvesafb: VBE get state call failed (eax=0x4f04, err=-7)
udev: starting version 158
EXT3-fs (hda): using internal journal
uvesafb: message too long (57808), can't execute task
uvesafb: VBE get state call failed (eax=0x4f04, err=-7)
uvesafb: message too long (57808), can't execute task
uvesafb: VBE get state call failed (eax=0x4f04, err=-7)
uvesafb: message too long (57808), can't execute task
uvesafb: VBE get state call failed (eax=0x4f04, err=-7)

JL 13/0/10

Index: linux/include/linux/connector.h
===================================================================
--- linux.orig/include/linux/connector.h	2010-08-13 13:16:28.000000000 +0100
+++ linux/include/linux/connector.h	2010-08-13 13:18:29.507791929 +0100
@@ -48,7 +48,8 @@
 /*
  * Maximum connector's message size.
  */
-#define CONNECTOR_MAX_MSG_SIZE		16384
+//#define CONNECTOR_MAX_MSG_SIZE		16384
+#define CONNECTOR_MAX_MSG_SIZE		65536
 
 /*
  * idx and val are unique identifiers which 
